### PR TITLE
request to add my domain

### PR DESCRIPTION
**Pull request by Muppidathi-sys**

### CODE REVIEW

File: domains/sathish.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My personal dev blog",
+  "repo": "https://github.com/Muppidathi-sys/sathish",
+  "owner": {
+    "username": "Muppidathi-sys",
+    "email": "muppidathi00@gmail.com",
+  },
+  "record": {
+    "CNAME": "Muppidathi-sys.github.io"
```
Comment (MaskDuck) [inline domains/sathish.json:9]: 404?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by omxpro** [comment]
Screenshot_20220418-171244_Brave

Btw aren't repo supposed to end with username.github.io

**Comment by MaskDuck** [comment]
theres a trick to name the repo whatever you like :)

**Comment by Muppidathi-sys** [comment]
Sorry, there was a mistake in mapping the repo to domain. It works now.
https://muppidathi-sys.github.io/

**Review by MaskDuck — DISMISSED** [review]

**Comment by Muppidathi-sys** [comment]
please verify. and merge quickly.

**Comment by MaskDuck** [comment]
why are you constantly closing and re-opening?

**Comment by Muppidathi-sys** [comment]
sorry bro, by mistake!!!

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**